<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Splash screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        QWidget
        {
            color: #fff;
            background-color: #01889C;
            selection-background-color:#3daee9;
            selection-color: #fff;
            background-clip: border;
            border-image: none;
            border: 0px transparent black;
            outline: 0;
        }

        QProgressBar {
            border: 1px solid #fff;
            border-radius: 5px;
            text-align: center;
        }

        QProgressBar::chunk {
            background-color: #05B8CC;
        }
    </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="splah_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>750</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>750</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>750</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
